rpc/src/calls/personal.rs

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+use accounts::{get_key_dir, Account};
+use client::BlockKey;
+use client::ValidatorClient;
+use jsonrpc_core::{Error, Params, Value};
+use messages::seth::{
+    CreateExternalAccountTxn, EvmPermissions, SethTransaction as SethTransactionPb,
+    SethTransaction_TransactionType,
+};
+use openssl::ec::{EcGroup, EcKey};
+use openssl::nid::Nid;
+use openssl::symm::Cipher;
```

Comment:
This should use the signing library to the extent possible: https://github.com/hyperledger/sawtooth-core/blob/master/sdk/rust/src/signing/secp256k1.rs

Reply:
It looks like the signing library can load pem files, but can't save/generate them, unless I'm misreading it?